frontend/src/metabase/forms/components/FormSubmitButton/FormSubmitButton.tsx

```diff
@@ -8,11 +8,13 @@ import { Button } from "metabase/ui";
 import type { FormStatus } from "../../contexts";
 import { useFormSubmitButton } from "../../hooks";
 
-export interface FormSubmitButtonProps extends Omit<ButtonProps, "children"> {
-  label?: string;
-  activeLabel?: string;
-  successLabel?: string;
-  failedLabel?: string;
+export interface FormSubmitButtonProps
+  extends Omit<ButtonProps, "children">,
+    Omit<HTMLAttributes<HTMLButtonElement>, "color"> {
```

Comment:
I _think_ ButtonProps already extends the HTMLButton type, so you might be able to make this `Omit<ButtonProps, "children" | "color">`